clar2wasm/src/wasm_generator.rs

```diff
@@ -589,6 +589,16 @@ impl WasmGenerator {
         offset: u32,
         ty: &TypeSignature,
     ) -> u32 {
+        const BYTES_COUNT: fn(&TypeSignature) -> u32 = |ty| {
```

Comment:
Can this logic replace the `get_type_size` instead of being a local closure?

Reply:
Yes, that's much better! No idea this function existed!